amp/TEMPLATE/reampv2/src/modules/geocoder/components/layout/panel/GeocoderPanel.jsx

```diff
@@ -1,23 +1,101 @@
 import React, {Component} from 'react';
 import {TranslationContext} from '../../AppContext';
 import GeocoderHeader from "./GeocoderHeader";
-import TableBootstrap from "../table/ActivityTable";
+import ActivityTable from "../table/ActivityTable";
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
+import GeocodingTable from "../table/GeocodingTable";
+import {SECTORS_CHART} from "../../../../sscdashboard/utils/constants";
 
-export default class GeocoderPanel extends Component {
+function GeocodingNotAvailable(props) {
+    return <h4>Geocoding process not available. User {props.user} is owner of the process in '{props.workspace}' workspace</h4>;
+}
+
+function GeocodingRunning() {
+    return (
+        <div>
+            <h4>Geocoding is still running</h4>
+        </div>
+    );
+}
+
```

Comment:
for functional component its clearer to use es6 notation 

const Hello = ({name}) => <div>Hello {name}</div>